src/applications/personalization/dashboard/tests/helpers.unit.spec.js

```diff
@@ -74,6 +76,23 @@ describe('profile helpers:', () => {
       });
     });
   });
+  describe('isFormAuthorizable', () => {
+    it('should return `true` if `authorize` is defined on the passed-in form config', () => {
+      const formConfig = {
+        authorize: () => {},
+      };
+      expect(isFormAuthorizable(formConfig)).to.be.true;
+    });
+    it('should return `false` if `authorize` is not defined on the passed-in form config', () => {
+      const formConfig = {};
+      expect(isFormAuthorizable(formConfig)).to.be.false;
+    });
+    it('should return `false` if it is passed in an undefined form config', () => {
+      let formConfig;
+      expect(isFormAuthorizable(formConfig)).to.be.false;
```

Comment:
this is the test that would fail with `Cannot read property 'authorize' of undefined` before I adjusted the `isFormAuthorizable()` helper to be safer.